public/style.css

```diff
@@ -29,6 +29,10 @@
 	color: #5d5b6d;
 }
 
+.bg-navbar{
+  color: #9B9B9B;
```

Comment:
the name of the class is background navbar, why not move the color to a different class ?